index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<head>
+  <title>Vega Lite Bar Chart</title>
+  <meta charset="utf-8">
+
+  <script src="http://d3js.org/d3.v3.min.js"></script>
+  <script src="http://vega.github.io/vega/vega.js"></script>
+  <script src="http://vega.github.io/vega-lite/vega-lite.js"></script>
+  <script src="http://vega.github.io/vega-editor/vendor/vega-embed.js" charset="utf-8"></script>
+  <script src="src/vl-tooltip.js" charset="utf-8"></script>
+  <link rel="stylesheet" type="text/css" href="src/vl-tooltip.css">
+
+  <style media="screen">
+    /* Add space between vega-embed links  */
+    .vega-actions a {
+      margin-right: 5px;
+    }
+  </style>
+
+  <script>
+  // Scatter Plot
+  var scatterSpec = {
+    "data": {"url": "data/cars.json"},
+    "mark": "circle",
+    "encoding": {
+      "x": {"field": "Horsepower", "type": "quantitative"},
+      "y": {"field": "Miles_per_Gallon", "type": "quantitative"}
+    }};
+  var scatterEmbed = {
+    mode: "vega-lite",  // Instruct Vega-Embed to use the Vega-Lite compiler
+    spec: scatterSpec
+  };
+  vg.embed("#vis-scatter", scatterEmbed, function(error, result) {
+    result.view.on("mouseover", fillTooltip);
```

Comment:
As a plugin, why don't you wrap these lines
```
result.view.on("mouseover", fillTooltip);
result.view.on("mouseout", clearTooltip);
```
in one function that takes `result` as an input.